eth/rpc/backend/utils.go

```diff
@@ -229,7 +231,7 @@ func AllTxLogsFromEvents(events []abci.Event) ([][]*gethcore.Log, error) {
 // TxLogsFromEvents parses ethereum logs from cosmos events for specific msg index
 func TxLogsFromEvents(events []abci.Event, msgIndex int) ([]*gethcore.Log, error) {
 	for _, event := range events {
-		if event.Type != evm.EventTypeTxLog {
+		if event.Type != proto.MessageName((*evm.EventTxLog)(nil)) {
```

Comment:
```suggestion
		if event.Type != proto.MessageName(new(evm.EventTxLog)) {
```

Technically equivalent, but nil pointers look a bit concerning